source/views/sis/course-search/results.js

```diff
@@ -83,8 +85,19 @@ class CourseSearchResultsView extends React.Component<Props, State> {
 		}
 	}
 
+	scrollResultsToTop = () => {
+		this.resultsList.scrollToLocation({
+			animated: false,
+			itemIndex: 0,
+			sectionIndex: 0,
+			viewOffset: Platform.OS === 'ios' ? 87 : 100,
```

Comment:
Are these rotationally dependent magic numbers?

Reply:
@drewvolz They shouldn’t be...It’s just the height of the filter toolbar, which is a little bigger on android, but shouldn’t change based on orientation..but I haven’t tested it so I can’t say for sure.

Reply:
What if we just scrolled all the way to `0`?

Reply:
@hawkrives You mean without the viewOffset? Then it cuts of& the FilterToolbar. I can post a screenshot if you want.

Reply:
Please 😄

Reply:
With | Without
-- | --
![with](https://user-images.githubusercontent.com/24640726/44219119-dfd42000-a140-11e8-833b-b59cac03c80d.gif) | ![without](https://user-images.githubusercontent.com/24640726/44219137-e95d8800-a140-11e8-8463-aa953e718a7f.gif)

Reply:
@drewvolz I have tested it now and they are not rotationally dependent numbers, since the height of the toolbar is the same in portrait and landscape.

Reply:
Wait, so using this value makes us actually scroll to above the filterbar?

Reply:
It makes us scroll to the position where the padding above and below the filterbar is equal.